main.go

```diff
@@ -53,6 +53,11 @@ func main() {
 
 	kiln.LogInfo.Printf("ImageCreator set up.")
 
+	clusterConfig, err := kiln.NewClusterConfig()
```

Comment:
I think we should support out of cluster kubernetes client like we do in enrober/dispatcher as well. See: https://github.com/30x/enrober/blob/staging/pkg/server/server.go#L83-L113